### PR TITLE
[ADD] l10n_nl: add missing stuff for the Dutch localization

### PR DESCRIPTION
**Pull request by smetl**
- Add new tags to have a more detailed P&L with sub-categories.
- Add Standard Business Reporting (SBR) codes and link them to l10n_nl coa.
- Add/delete some accounts to the l10n_nl coa.

see task: https://www.odoo.com/web#id=31825&view_type=form&model=project.task&action=333&active_id=967&menu_id=4720

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by qdp-odoo** [comment]
as discussed, remove the part related to the SBR for now: we might use the accuont.group feature instead (difficult to say for now as there isn't any feature using it yet!)

File: addons/l10n_nl/data/account_account_tag.xml [hunk 1]
```diff
@@ -1,54 +1,138 @@
...
+            <field name="name">Vorderingen</field>
             <field name="applicability">accounts</field>
         </record>
         <record id="account_tag_6" model="account.account.tag">
-            <field name="name">Auto</field>
+            <field name="name">Effecten</field>
             <field name="applicability">accounts</field>
         </record>
         <record id="account_tag_7" model="account.account.tag">
-            <field name="name">Verkoopkosten</field>
```
Comment (StefanRijnhart) [inline addons/l10n_nl/data/account_account_tag.xml]: You rename account_tag_7 from Verkoopkosten to Liquide middelen and then create a new tag account_tag_25 Verkoopkosten. This seems like a migration nightmare for poor @odony or whoever is doing that these days. How about keeping account_tag_7 as it is and add Liquide middelen as a new tag? There are other examples of this as well.

Comment (smetl) [inline addons/l10n_nl/data/account_account_tag.xml]: @StefanRijnhart Ok. I swapped:

1 <-> 17
2 <-> 23
5 <-> 27
6 <-> 24
7 <-> 25
8 <-> 28
10 <-> 15
11 <-> 14
12 <-> 32

Comment (StefanRijnhart) [inline addons/l10n_nl/data/account_account_tag.xml]: Thanks!

**Comment by qdp-odoo** [comment]
this one is ok

**Comment by qdp-odoo** [comment]
landed in master, for saas-18, at revisions:
- https://github.com/odoo/odoo/commit/c75fe95dfca366bf6c3b6a51908677039488aa4f
- https://github.com/odoo/enterprise/commit/998d8a15fad88a3e48af206f1ee01fe0d0522234

thanks

**Pull request closed without merging**